csuntechlab/takeoff
# Setup docs

# Description
Added a /docs directory which is powered by VuePress. The docs are intended to be a living document, providing one source of truth for the project, maintained by developers/designers/marketers.

Also updated some of the front-end tooling (import bootstrap scss files independently, setup some base color variables based on style tiles, install fontawesome 5 instead of 4).
  
# Testing
1) run `yarn` in order to install new dependencies. *note: if you get errors, try deleting your entire `node_modules` directory and run `yarn` again.
2) run `yarn dev` and `yarn prod` commands to ensure you don't get any errors
3) run `yarn docs:dev` and ensure you don't get any errors. Try to make a minor change to some of the text on the docs.
4) run `yarn docs:build` and ensure you don't get any errors. Then see if your minor change was reflected when viewing the /docs directory through docker/valet/mamp.
